requirements.txt

```diff
@@ -1,4 +1,2 @@
 pip>=21.1
-poetry==1.1.15
-tox>=3.25.0
-tox-poetry>=0.4.1
+poetry
```

Comment:
IMHO pin it to something you know. Maybe poetry==1.4.0 if that's what you're now working on.

I'm a believer that base tool requirements should be pinned.

Reply:
That's the latest?